src/TabsPage.tsx

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import classNames from 'classnames';
+import { Link, useParams } from 'react-router-dom';
+import { Tab } from './types/Tab';
+
+interface Props {
```

Comment:
Similar names are okay for a small project, but on large projects we will have many interfaces and types, so it is better to give more detailed names, for example `TabsPageProps`.
It would also be really cool to keep similar types in one place, so you could move this type to the `types.ts` file and use only ready-made imports here. This would shorten the code a bit and make it cleaner.